### DISCUSSION TITLE
Conditional include

### DISCUSSION THREAD

**Discussion by ArtsiomChyrkun** [prisma/prisma#14948] [Q&A] [answered]
Hi, I have a kinda weird use case. I have a business entity which can have a status chosen by an user. I want to know if it's possbile to get a list of businesses, but include a status only for a specific user if it has a status with this specific user_id?
If I simply put `include: { status: true }` it will return all statuses for all users.

My model:
```
model business {
  business_id      String            @id
  business_name    String
  business_type    String
  abn              Json
  gst              Json
  image_url        Json
  dob              DateTime          @db.Date
  website          String?
  full_address     String?
  primary_category String
  employee         Int
  revenue          Int?
  category         category          @relation(fields: [primary_category], references: [cat_code])
  mails            business_mail[]
  phones           business_phone[]
  socials          business_social[]
  coordinates      Json?
  status           business_status[]

  @@map("business_business_detail")
}

model business_status {
  id          String   @id @default(uuid()) @db.Uuid
  business_id String
  user_id     String
  status      String
  business    business @relation(fields: [business_id], references: [business_id])

  @@unique([business_id, user_id])
  @@map("business_business_status")
}
```
Thanks in advance.

**Comment by austincrim** [comment, marked as answer]
Hey @ArtsiomChyrkun 👋,

You could try something like this: 
```ts
let result = await prisma.business.findMany({
  include: {
    status: {
      where: {
        user_id: 'foo'
      }
    }
  }
})
```

**Reply by ArtsiomChyrkun** [reply]
Thank you, it works. I was inattentive reading the doc.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
